modules/loki-deployment-sizing.adoc

```diff
@@ -13,59 +13,69 @@ endif::[]
 Sizing for Loki follows the format of `<N>x.<size>` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
 
 .Loki sizing
-[cols="1h,3*",options="header"]
+[cols="1h,4*",options="header"]
 |===
 |
+|1x.demo
 |1x.extra-small
 |1x.small
 |1x.medium
 
 |Data transfer
+|Demo use only
 |100GB/day
 |500GB/day
 |2TB/day
 
 |Queries per second (QPS)
+|Demo use only
 |1-25 QPS at 200ms
 |25-50 QPS at 200ms
 |25-75 QPS at 200ms
 
 |Replication factor
+|None
 |2
 |2
 |2
 
 |Total CPU requests
+|None
 |14 vCPUs
 |34 vCPUs
 |54 vCPUs
 
 ifdef::restricted[]
 |Total CPU requests if using the ruler
+|None
 |16 vCPUs
 |42 vCPUs
 |70 vCPUs
 endif::restricted[]
 
 |Total memory requests
+|None
 |31Gi
 |67Gi
 |139Gi
 
 ifdef::restricted[]
 |Total memory requests if using the ruler
+|None
 |35Gi
 |83Gi
 |171Gi
 endif::restricted[]
 
 |Total disk requests
+|None
 |430Gi
 |430Gi
 |590Gi
 
 ifdef::restricted[]
 |Total disk requests if using the ruler
+|None
 |650Gi
 |650Gi
```

Comment:
it is 750Gi

Reply:
```
oc get pvc  |awk '{print $4 "   " $1}'
CAPACITY   NAME
10Gi   storage-lokistack-sample-compactor-0
50Gi   storage-lokistack-sample-index-gateway-0
50Gi   storage-lokistack-sample-index-gateway-1
10Gi   storage-lokistack-sample-ingester-0
10Gi   storage-lokistack-sample-ingester-1
10Gi   storage-lokistack-sample-ruler-0
10Gi   storage-lokistack-sample-ruler-1
150Gi   wal-lokistack-sample-ingester-0
150Gi   wal-lokistack-sample-ingester-1
150Gi   wal-lokistack-sample-ruler-0
150Gi   wal-lokistack-sample-ruler-1
```

Reply:
@anpingli it seems odd that this value for the demo is larger than the supported-in-production sizes. Do the other sizes also need to be updated? Also, would this mean that without the ruler, the disk reqs for the demo size is 450Gi?

Reply:
@abrennan89   750Gi is for extra-small and small.    it is not for demo.

Reply:
for demo:  the total pvc size is 40Gi without ruler. it is 80Gi with ruler.

Reply:
That makes a lot more sense, thanks